frontend/src/components/program/picasso/Picasso.vue

```diff
@@ -291,6 +290,7 @@ export default {
         ...entry,
         startTimestamp: utcStringToTimestamp(entry.start),
         endTimestamp: utcStringToTimestamp(entry.end),
+        duration: minuteDifference(entry.end, entry.start),
```

Comment:
This is not reactive. At the moment kind of working, because this is recalculated after network call has finished. Wouldn't it be better we put duration into a computed?

Might make it necessary to move the content for a single schedule entry into a separate component.

Reply:
I beliefe this is made reactive: https://github.com/usu/ecamp3/pull/9/files/636e1482ae172fe1f009d7e29da72476eadf436a#diff-7d3583d2b82ab1f16c2586ed653cee877f32a42a3b9a157da8bc804d8b70c713R303

Reply:
> I beliefe this is made reactive:

No, it's not 👉 Resize the scheduleEntry, the icon size will not change until there is a network request which is finished